.github/workflows/publish.yml

```diff
@@ -98,5 +104,6 @@ jobs:
           GH_TOKEN: ${{ github.token }}
         working-directory: build
         run: |
-          Compress-Archive -Path "bmx" -DestinationPath "bmx-$env:PLATFORM.zip"
-          gh release upload "$env:RELEASE_TAG" "bmx-$env:PLATFORM.zip"
+          chmod u+x "bmx"
+          tar -cvf "bmx-$env:PLATFORM.tar" "bmx"
+          gh release upload "$env:RELEASE_TAG" "bmx-$env:PLATFORM.tar"
```

Comment:
when PowerShell isn't needed, there's merit to using bash due to its more reliable fail fast behaviour